ssh.c

```diff
@@ -1129,6 +1129,10 @@ main(int ac, char **av)
 		}
 	} else {
 		/* A command has been specified.  Store it into the buffer. */
+		if (ac > 1)
+			fprintf(stderr,
+				"Warning: arguments given to command. "
+				"Concatenating with spaces.\n");
```

Comment:
No sorry - many, many people use ssh with commandline arguments every day both manually and in scripts. This would just be noise.

Reply:
I don't disagree that many people use it, but to me it still is a big footgun with a severity ranging from "use this example to scare newbies" to "throw semicolons at it and you might get a command injection".

There's also the more modest approach mentioned in the mailing list: we expand the manual page for examples of this stuff used "responsibly", whatever that could mean. That will take much more writing and is less visible.

Reply:
We *could* do some extra scanning to avoid the basic `ssh x@x sh /etc/xxx` use cases. Like look for spaces, which is the most common failure case; or something like `;|&` which can launch extra commands. But simple scanning will still flag specially quoted arguments and "just use a single `command` parameter" would be a less obvious solution compared to what this message implies.